packages/server/database/types/MSTeamsNotifications.ts

```diff
@@ -0,0 +1,40 @@
+import generateUID from '../../generateUID'
+
+export const MSTeamsNotificationEventTypeLookup = {
+  meetingStart: 'team',
+  meetingEnd: 'team',
+  MEETING_STAGE_TIME_LIMIT_END: 'member',
+  MEETING_STAGE_TIME_LIMIT_START: 'team'
+} as const
+
+export type MSTeamsNotificationEvent = keyof typeof MSTeamsNotificationEventTypeLookup
+export type MSTeamsNotificationEventEnum =
```

Comment:
-1 There is no reason to have this defined separately for MS Teams, we can just use the `SclackNotificationEventEnum` as it's done in Mattermost. We will move and rename it eventually.